requirements/requirements-docs.txt

```diff
@@ -3,6 +3,7 @@ ipykernel~=6.5
 nbconvert~=6.5.1
 nbsphinx~=0.8.8
 notedown
+lxml~=5.1.0
```

Comment:
Newly released `lxml-5.2.0` breaks the doc build process, need to cap the dependency